### ISSUE TITLE
openmediavault.postinst fails due to chronyd refusing to start on systems running a kernel with SECCOMP disabled

### ISSUE THREAD

**Issue by dleidert** [dleidert/openmediavault-upgrade#3]
This is about the following error:

```
[..]
Setting up Salt environment ...
Processing system modifications ...
dpkg: error processing package openmediavault (--configure):
 installed openmediavault package post-installation script subprocess returned error exit status 1

Errors were encountered while processing:
 openmediavault
```

due to this line in `openmediavault.postinst`:

```
omv-salt deploy run --no-color --quiet collectd chrony postfix
```

Resources for this issue so far:

<https://forum.openmediavault.org/index.php?thread/31985-solved-openmediavault-package-post-installation-script-subprocess-error/>
<https://forum.openmediavault.org/index.php?thread/32518-chronyd-fails-to-start-service-chronyd-4458-fatal-error-failed-to-load-seccomp-r/>
<https://gist.github.com/jjakob/c9ab333d15105b986008863e9f4f5db8>

**Comment by dleidert** [comment]
<https://forum.openmediavault.org/index.php?thread/37269-upgrade-scripts-for-non-interactive-major-release-upgrades-2-3-3-4-4-5/&postID=260735#post260735>

**Comment by dleidert** [comment]
Fixed via fb274dc
